phoenix-core/src/test/java/org/apache/phoenix/expression/InListExpressionTest.java

```diff
@@ -19,13 +19,19 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
 
 import com.google.common.collect.Lists;
 import java.util.ArrayList;
 import java.util.List;
+
 import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.phoenix.compile.WhereOptimizer;
 import org.apache.phoenix.hbase.index.util.ImmutableBytesPtr;
+import org.apache.phoenix.schema.PColumn;
+import org.apache.phoenix.schema.PTable;
 import org.junit.Test;
+import org.mockito.Mockito;
 
 public class InListExpressionTest {
```

Comment:
The tests for this should not live in this class.  This class should be only for InListExpression changes, of which you have none.